### PR TITLE
Microsites changes

### PR DESCRIPTION
**Pull request by LGLO**
*~Moving background to homepage~
* Removing redundant Overview in Overview menu
* Using `home` layout instead of `page`
* Removed stray social media links

### CODE REVIEW

File: microsite/src/main/tut/overview/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs
-position: 2
+position: 3
```
Comment (LGLO) [inline microsite/src/main/tut/overview/index.md]: Getting Started is 2nd, this is 3rd.

**Comment by jdegoes** [comment]
@LGLO Can you include screenshots of the changes? Thank you!

**Comment by LGLO** [comment]
Getting started gets same layout as Home, currently it's different - take a look at site.
GettingStarted

Overview has fixed menu. Also Twitter, FB, Google+ icons are removed
: Overview 2019-04-07 21-59-34

**Comment by jdegoes** [comment]
@LGLO Looks great! thank you and @wi101!

**Pull request merged**